firehose-extract/build.rs

```diff
@@ -1,8 +1,14 @@
+use std::fs;
 use std::io::Result;
+
 fn main() -> Result<()> {
     prost_build::compile_protos(
         &["type.proto"],
         &["../firehose-fuel/proto/sf/fuel/type/v1/"],
     )?;
+
+    fs::write("target/schema.sdl", fuel_core_client::SCHEMA_SDL)
```

Comment:
Hmm, you can publish `fuel-core-client-ext` and use it directly instead of duplicating the code=)

Reply:
https://github.com/FuelLabs/fuel-core-client-ext/pull/1

Reply:
https://github.com/FuelLabs/fuel-core-client-ext/pull/2

Reply:
Ok done